app/services/album.js

```diff
@@ -22,3 +22,35 @@ exports.getPhotosOfAlbum = albumId => {
       throw errors.conectionError(e.message);
     });
 };
+
+exports.getAllAlbums = (offset, limit, orderBy) => {
+  const endpoint = `${url}albums`;
+  return axios
+    .get(endpoint)
+    .then(response => response.data)
+    .catch(e => {
+      throw errors.conectionError(e.message);
+    })
+    .then(albums => {
+      if (orderBy) {
+        const slicedAlbums = albums
+          .slice()
+          .sort((a, b) => {
+            if (!a[orderBy] || !b[orderBy]) {
+              throw errors.badRequest('The orderBy parameter do not exist');
+            }
+            if (a[orderBy] < b[orderBy]) {
+              return -1;
+            }
+            if (a[orderBy] > b[orderBy]) {
+              return 1;
+            }
+            return 0;
+          })
+          .slice(offset, limit);
+        return slicedAlbums;
+      }
+      const slicedAlbums = albums.slice(offset, limit);
```

Comment:
and what if you have both order and limit? can you handle them with this?

Reply:
Yes, after the sort I add the slice too